ui/components/app/index.jsx

```diff
@@ -48,11 +48,22 @@ export default class App extends translatable(Component){
       <aside className="col-xs-4 col-md-3 col-lg-2">
         <div className="row">
           <div role="navigation">
-               {navigationLink(this.__("Overview"), 'search', tabs.OVERVIEW)}
-               {navigationLink(this.__("Location"), 'map-marker', tabs.PLANNING)}
-               {navigationLink(this.__("Competitiveness"), '', tabs.COMPETITIVENESS)}
-               {navigationLink(this.__("Efficiency"), '', tabs.EFFICIENCY)}
-               {navigationLink(this.__("e-Procurement"), '', tabs.E_PROCUREMENT)}
```

Comment:
Hey Walter, 
This was a temporary implementation. 
Thank you!
Gali

Sent from my iPhone

> On Jul 18, 2016, at 1:49 PM, Schneider Werner-Walter <notifications@github.com> wrote:
> 
> In ui/components/app/index.jsx:
> 
> > @@ -48,11 +48,22 @@ export default class App extends translatable(Component){
> >        <aside className="col-xs-4 col-md-3 col-lg-2">
> >          <div className="row">
> >            <div role="navigation">
> > -               {navigationLink(this.__("Overview"), 'search', tabs.OVERVIEW)}
> > -               {navigationLink(this.__("Location"), 'map-marker', tabs.PLANNING)}
> > -               {navigationLink(this.__("Competitiveness"), '', tabs.COMPETITIVENESS)}
> > -               {navigationLink(this.__("Efficiency"), '', tabs.EFFICIENCY)}
> > -               {navigationLink(this.__("e-Procurement"), '', tabs.E_PROCUREMENT)}
> Instead of replacing the component with custom HTML markup you should update the navigationLink() component.
> 
> For changing the navigationLink markup: https://github.com/devgateway/ocvn/blob/develop/ui/components/app/navigation-link/index.jsx
> 
> For adding an extra Icon parameter see: https://github.com/devgateway/ocvn/blob/develop/ui/components/app/index.jsx#L25-L26
> 
> ―
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>